lib/diagnostic.js

```diff
@@ -14,12 +14,30 @@
 'use strict';
 
 const fs = require('fs'); // jshint ignore: line
+const stdin = '/dev/stdin';
 
-const sumLines = (filename, callback) => {
-  /* your response here */
-  callback();
+let inFile = process.argv[2] === '-' ? stdin : process.argv[2];
+
+const sumLines = (array /*filename, callback*/) => {
```

Comment:
generally speaking, it is NOT a good idea to delete the code provided to you in a diagnostic. Without the 2nd argument, your code will not be able to run according to the instructions provided above. Notice that it says, "It should add all the numbers and pass the result as the second argument to the callback provided. e.g `callback(null, sum);`.